doc/source/_toc.yml

```diff
@@ -105,6 +105,21 @@ parts:
           - file: workflows/package-ref
       - file: ray-more-libs/index
         title: More Ray ML Libraries
```

Comment:
Can we create an `images` folder inside of `ray-air`? I think that's cleaner (we have too many images in there already, not clear what belongs where)